src/App.tsx

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedButton: string | boolean;
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    pressedButton: false,
```

Comment:
I'd like to change ` pressedButton: ''`, that will be `false` as boolean value, but up to you!)

Reply:
That's useful. Thanks! 😊